scripts/submit_sim_imcalc.py

```diff
@@ -124,9 +127,7 @@ def submit_im_calc_slurm(
                     "got more than one folder globbed. please double check the path to the match_obs_stations is correct."
                 )
                 sys.exit()
```

Comment:
Will this exit close autosubmit too? If so, should print something to the console too.

Reply:
logger.error prints to the console, as well as to the log file

Reply:
Easy!